handlers/api.py

```diff
@@ -8,28 +8,29 @@
 from googleapiclient.http import *
 
 _SCOPES = ['https://www.googleapis.com/auth/drive']
+_token = '../credentials/token.pickle'
```

Comment:
[Optional]
Usually when some values are hardcoded in a module outside of class, they are constants and it's good practice to name them upper-case. Also, I think you should not use underscores here, as it's not a class.